tests/install/conftest.py

```diff
@@ -80,6 +80,13 @@ def iso_remaster(request, answerfile):
 
     gen_unique_uuid = marker.kwargs.get("gen_unique_uuid", False)
 
+    if source_type == "pxe":
+        # ISO remastering is not needed when booting with PXE so we just
+        # return
+        logging.info("iso_remaster not needed with PXE")
+        yield None
+        return
+
```

Comment:
We still need a number of things which currently live here (and so should arguably live somewhere else):
- ssh authorized keys in `install.img`
- generation of postinstall script (and providing it with `test-pingpxe` and `test-unique-uuids` services to install on host)
- generation of answerfile